### PR TITLE
Chapter 11

### PR DESCRIPTION
**Pull request by balqis277**
ready

### CODE REVIEW

**Comment by balqis277** [comment]
ready

File: app/assets/stylesheets/account_activations.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the AccountActivations controller here.
```
Comment (anlvq-1058) [inline app/assets/stylesheets/account_activations.scss:1]: delete this file please

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -1,13 +1,19 @@
 class SessionsController < ApplicationController
-  def new; end
+  def new
+  end
```
Comment (anlvq-1058) [inline app/controllers/sessions_controller.rb:3]:
```suggestion
  def new; end
```

File: app/helpers/account_activations_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
```
Comment (anlvq-1058) [inline app/helpers/account_activations_helper.rb:1]: delete this file please

File: test/controllers/account_activations_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (anlvq-1058) [inline test/controllers/account_activations_controller_test.rb:1]: delete this file please

File: test/mailers/previews/user_mailer_preview.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user_mailer
+class UserMailerPreview < ActionMailer::Preview
```
Comment (anlvq-1058) [inline test/mailers/previews/user_mailer_preview.rb:2]: delete this file please

File: test/mailers/user_mailer_test.rb [hunk 6]
```diff
@@ -0,0 +1,20 @@
+require "test_helper"
+
+class UserMailerTest < ActionMailer::TestCase
+  test "account_activation" do
+    mail = UserMailer.account_activation
+    assert_equal "Account activation", mail.subject
+    assert_equal ["to@example.org"], mail.to
```
Comment (anlvq-1058) [inline test/mailers/user_mailer_test.rb:7]: delete this file please

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by balqis277** [comment]
ready

**Comment by balqis277** [comment]
ready

File: app/controllers/users_controller.rb [hunk 7]
```diff
@@ -59,9 +53,17 @@ def destroy
 
   private
 
+  def correct_user
+    @user = User.find(params[:id])
+    return if @user==current_user?(@user)
```
Comment (anlvq-1058) [inline app/controllers/users_controller.rb:58]:
```suggestion
    return if @user == current_user?(@user)
```

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by balqis277** [comment]
ready

**Comment by balqis277** [comment]
ready

**Review by anlvq-1058 — COMMENTED** [review]
merged

**Pull request merged**